apps/auction2/components/auction_grid/index.js

```diff
@@ -0,0 +1,18 @@
+import { default as React, PropTypes } from 'react';
+import AuctionGridArtwork from '../auction_grid_artwork/index'
+
+export default function AuctionGrid({ artworks }, _) {
```

Comment:
Also lint catchable but this underscore as a second argument (for React context) is unnecessary. Also if you want to win code golf you could use arrow functions `export default ({ artworks }) => {`... however, then you give up `propTypes` which I guess is also a debatable style choice. I don't have a strong opinion on this, just pointing it out the options.